src/metabase/search/postgres/scoring.clj

```diff
@@ -53,18 +53,42 @@
    [[(sum-columns (map weighted-score scorers))
      :total_score]]))
 
+;; Divides rank by log(len(doc))
+;; See https://www.postgresql.org/docs/current/textsearch-controls.html#TEXTSEARCH-RANKING
+(def ^:private ts-rank-normalization 1)
+
+(def ^:private bookmarked-models [:card :collection :dashboard])
+
+(def ^:private bookmark-score-expr
+  (let [match-clause (fn [m] [[:and [:= :model m] [:!= nil (keyword (str m "_bookmark." m "_id"))]]
```

Comment:
Should maybe `bookmarked-models` be a map to a field name instead of string concat here?

Reply:
Good idea